contrib/containers/ci/fedora-30.docker

```diff
@@ -0,0 +1,4 @@
+FROM fedora:30
```

Comment:
I like the centos/rhel as they are not likely to change frequently, but how about replacing the fedora-30 with fedora as they will have to be frequently updated?